Makefile

```diff
@@ -165,9 +168,14 @@ build: generate fmt vet ## Build manager binary.
 run: manifests generate fmt vet ## Run a controller from your host.
 	go run ./main.go
 
-docker-build:
+docker-build-operator:
 	docker buildx build --load -t ${IMG} .
 
+docker-build-secrets-webhook:
```

Comment:
Would it possibly be worth importing the makefile targets from the webhook's makefile into this makefile? You might need to change their names to avoid collisions, but right now you'd need to `cd` into the webhook folder, build the binary, then cd out before building everything else.